server/db/models/Product.js

```diff
@@ -1,12 +1,48 @@
 const { STRING } = require('sequelize')
 const db = require('../db')
 
+const Product = conn.define('product', {
+    title: {
+        type: STRING,
+        allowNull: false,
+        validate: {
+            notEmpty: true,a
+        },
+    },
+    brand: {
+        type: STRING,
+        allowNull: false,
+    },
+    category: {
+        type: STRING,
+        validate: {
+            isIn: [['sweet', 'salty', 'healthy']],
```

Comment:
We might have more than these categories so I think we don't need this validation (line 18 - 19). Just do allowNull: false

Reply:
cant we omit this field all together i though that category was an associated table. the foreign keys would handle this id assume?

Reply:
Hmm, I sure can just do allowNull: false right now...I was thinking about maybe we can add 'frozen' or 'drink' as well...It's up to you @alejvillanueva really since you will be in charge of the category module.@kfless12 I think for now I will let Alejandra decide what she wants to do, and maybe we can talk more about it tomorrow!

Reply:
Oh, Kevin is right! That detailed slipped my mind. When we do the relationships, the categoryId will be generated.